plugins/nginx-vhosts/commands

```diff
@@ -50,7 +50,7 @@ case "$1" in
     fi
 
     DOKKU_APP_CIDS=($(get_app_container_ids $APP))
-    docker cp "${DOKKU_APP_CID[0]}:/app/nginx.conf.template" "$APP_NGINX_TEMPLATE" 2> /dev/null || true
+    docker cp "${DOKKU_APP_CIDS[0]}:/app/nginx.conf.template" "$DOKKU_ROOT/$APP/" 2> /dev/null || true
```

Comment:
As I tested it - no. You should define only directory where template should be stored. If you use $APP_NGINX_TEMPLATE as the full path, docker will create directory with this name, not the file.